addons/mail/static/src/owl/components/chatter/chatter.scss

```diff
@@ -0,0 +1,29 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Chatter {
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+    width: 530px;
+    padding: 0;
+}
+
+.o_Chatter_composerContainer {
+    position: relative;
+}
```

Comment:
Do you need container for that? Couldn't make `.o_Chatter_composer` with `position: relative`?